backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0005_AddRegionApplicationActivation.kt

```diff
@@ -0,0 +1,20 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import app.ehrenamtskarte.backend.migration.Statement
+
+/**
+ * Add firstActivationDate column to cards.
+ * Set current_timestamp for firstActivationDate for existing active cards since we have no firstActivationDate for them.
+ */
+@Suppress("ClassName")
+internal class V0005_AddRegionApplicationActivation() : Migration() {
+    override val migrate: Statement = {
+        exec(
+            """
+            ALTER TABLE regions 
+            ADD "activatedForApplication" BOOLEAN NOT NULL DEFAULT true;
```

Comment:
yes because daniel said we can not affort that every region that wants to use the application process has to activate it. Therefore there is a second ticket for a sql statement that deactivates the regions that don't want to start with